decidim-proposals/config/locales/en.yml

```diff
@@ -60,6 +61,12 @@ en:
             evaluating: Evaluating
             rejected: Rejected
             title: Answer for proposal %{title}
+        proposal_notes:
+          form:
+            comment: Comment
+            submit: Submit
+          leave_your_comment: Leave your comment
```

Comment:
Shouldn't this read notes?

Reply:
Maybe it would be better: Leave your note